taskcluster/ci/config.yml

```diff
@@ -105,12 +105,6 @@ taskgraph:
             default-repository: https://github.com/mozilla-rally/rally-core-addon
             default-ref: release
             type: git
-        telemetrytestingextension:
-            name: "Telemetry testing extension"
-            project-regex: telemetrytestingextension$
-            default-repository: https://github.com/mozilla-extensions/dynamic-probe-telemetry-extension
-            default-ref: main
-            type: git
```

Comment:
there are more to remove from here as well as per chat yesterday- we have files in manifest/* that correspond to the entries in here (telemetrytesting was already removed) - I would be ok re-reviewing or reviewing a different followup patch.

Reply:
Yeah, I was thinking another patch for removing the rest